<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>monofur</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Workload</string>
  </property>
  <property name="windowOpacity">
   <double>0.900000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="taskInput"/>
    </item>
    <item>
     <widget class="QTreeWidget" name="taskList">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>monofur</family>
       </font>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="autoExpandDelay">
       <number>-1</number>
      </property>
      <property name="indentation">
       <number>10</number>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>2</number>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="headerCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>26</number>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>15</number>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Priority</string>
       </property>
       <property name="textAlignment">
        <set>AlignJustify|AlignVCenter</set>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Task</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>190</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTask">
    <property name="title">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuContext">
    <property name="title">
     <string>Context</string>
    </property>
    <addaction name="actionDefault"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTask"/>
   <addaction name="menuContext"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
